Pivot unpivot (#66)
implydata/learn-druid: notebooks/README.md

|[Using joins effectively in Druid](./03-query/11-joins.ipynb)|A full review of all join strategies available in Druid with examples and performance comparisons.|`druid-jupyter`|
|[Window functions (Experimental)](./03-query/13-query-functions-window.ipynb)|An introduction to Window functions which are a new experimental feature in Druid SQL.|`druid-jupyter`|
|[Query from Deep Storage](./03-query/14-full-timeline-queries.ipynb)|Query from Deep Storage has been enhanced to also view real-time segments making it capable of spanning the whole timeline.|`all-services`| 

> |[PIVOT and UNPIVOT functions](./03-query/15-pivot-unpivot.ipynb)|Convert a columns row values into independent columns with aggregate metrics using PIVOT. Use UNPIVOT to convert multiple columns into rows using UNPIVOT.|`druid-jupyter`| 

Reviewer: ```suggestion
|[PIVOT and UNPIVOT functions](./03-query/15-pivot-unpivot.ipynb)|Use PIVOT to convert row values into columns. Use UNPIVOT to convert column values into rows.|`druid-jupyter`| 
```
